rainier-core/src/main/scala/com/stripe/rainier/compute/Target.scala

```diff
@@ -0,0 +1,26 @@
+package com.stripe.rainier.compute
+
+class Target(val real: Real, val placeholders: Map[Variable, Array[Double]]) {
+  def inlined = inl(placeholders)
+
+  def inl(data: Map[Variable, Array[Double]]): Real =
+    if (data.isEmpty)
+      real
+    else {
+      val inlined = 0.until(nRows(data)).map { i =>
```

Comment:
A code comment similar to your pr description would help others and future-me follow this.

Reply:
Yeah I plan to add docs at the end of this whole process.